platform/conf/conf.go

```diff
@@ -40,7 +40,7 @@ type Conf struct {
 func New(userdata string) (*Conf, error) {
 	c := &Conf{}
 
-	ignc, err := v2.Parse([]byte(userdata))
+	ignc, _, err := v2.Parse([]byte(userdata))
```

Comment:
I think we'd want to log the report if there's an err for the convenient of the test author.